### ISSUE TITLE
Brew package for preview version?

### ISSUE THREAD

**Issue by gldtn** [zed-industries/zed#4373]
### Check for existing issues

- [X] Completed

### Describe the feature

I see that Zed is available on Brew: https://formulae.brew.sh/cask/zed

Please add brew package for preview version. It's makes more since to have in brew so we can keep up with the preview version that is constantly upgraded.

Would be nice if we can assign something like this or similar:

```
brew cask install zed --DEV
```

Thanks in advance!

**Comment by CurbaiCode** [comment]
Just FYI, `zed-preview` was added as a conflicting cask with the stable `zed`, so the installation command should most likely be:

    brew install --cask zed-preview

Though the preview cask is not currently available.

**Comment by notpeter** [comment]
This has been available since 2024-05-01 (Zed Preview v0.134.0) as [zed@preview](https://formulae.brew.sh/cask/zed@preview).  It does not conflict with the normal zed release (cli is aliased to `zed-preview` instead of `zed).

Install with:
```
brew install --cask zed@preview
```
